### PR TITLE
Refactor commercial E2E test structure

### PR DESCRIPTION
**Pull request by chrislomaxjones**
## What does this change?

- Remove unnecessary logging
- Each slot gets its own explicit test(s)
- Rename `slots.spec.ts` to `inline-slots.spec.ts`

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Clearer test structure. Will also allow us to have more custom logic for specific slots in the future.

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: cypress/integration/inline-slots.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { liveblogs, articles } from '../fixtures/pages';
+
+const pages = [articles[2], ...liveblogs];
```
Comment (domlander) [inline cypress/integration/inline-slots.spec.ts:3]: Could this be expanded to include any article where `expectedMinInlineSlotsOnPage` exists as a property? I'm worried we'll change the order of the articles in `fixtures/pages.ts` and this test will break

**Review by domlander — APPROVED** [review]

File: cypress/integration/inline-slots.spec.ts (hunk 1 above)
Comment (arelra) [reply cypress/integration/inline-slots.spec.ts]: Good point! We could filter on articles where that property exists?

Comment (zekehuntergreen) [reply cypress/integration/inline-slots.spec.ts]: Good call

**Review by Jakeii — APPROVED** [review]

**Review by arelra — APPROVED** [review]
🎉

File: cypress/integration/inline-slots.spec.ts (hunk 1 above)
Comment (chrislomaxjones) [reply cypress/integration/inline-slots.spec.ts]: Nice, thanks for adding @zekehuntergreen

**Pull request merged**